src/app.js

```diff
@@ -1,21 +1,25 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const { getFormattedPath } = require('./getFormattedPath');
+const { getResponse } = require('./getResponse');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
```

Comment:
redundant variable just return
```suggestion
  return http.createServer((req, res) => {
```